### ISSUE TITLE
Change sio:session redis key

### ISSUE THREAD

**Issue by ncanis** [socketio/socket.io-redis-streams-adapter#9]
I want to save it with the specified redis key, but there is no place to set it. Is it possible?

this.#redisClient.set(`sio:session:${session.pid}`, encodedSession ..
const sessionKey = `sio:session:${pid}`;

**Comment by darrachequesne** [comment]
Hi! Would a `sessionKeyPrefix` option (which would default to `"sio:session:"`) suit your needs?

**Comment by ncanis** [comment]
yes. I would like to override the redis key used in the library externally as shown below.
Since there are cases where permissions are assigned to each account for each specific Redis key, it is necessary to modify this RedisKey.

ex>
    // redis stream adapter
    const streamOption = {
      sioName: 'war2:dev',
      streamName: 'war2:dev:socket:stream`,
      maxLen: 100000,
      readCount: 1000,
    };`


this.#redisClient.set(`${this.#opts.sioName}:sio:session:${session.pid}`, encodedSession, {

this.#redisClient.xAdd(this.#opts.streamName, '*', RedisStreamsAdapter.encode(message), {

const entries = await this.#redisClient.xRange(this.#opts.streamName, RedisStreamsAdapter.nextOffset(offset), '+', {

**Comment by darrachequesne** [comment]
This was implemented in https://github.com/socketio/socket.io-redis-streams-adapter/commit/18985863d8eca842c59fa57e396a72c01123e61d, and included in version [`0.2.0`](https://github.com/socketio/socket.io-redis-streams-adapter/releases/tag/0.2.0).

Syntax:

```js
const io = new Server({
  adapter: createAdapter(redisClient, {
    sessionKeyPrefix: "some-custom-prefix:"
  })
});
```
